.circleci/main.yml

```diff
@@ -1923,33 +1933,49 @@ workflows:
             - aws-prerelease-testing
 
 commands:
-  install-python-test-deps: # Requires build-docker-images and build-pachctl-bin
+  install-helm:
     steps:
-      - attach_workspace:
-          at: ./
-      - run: sudo apt-get update && sudo apt-get install fuse libgbm-dev
-      - run:
-          name: Fuse options
-          command: echo "user_allow_other" | sudo tee -a /etc/fuse.conf > /dev/null
-      - run:
-          name: Install kind
-          command: |
-            curl -fLo ./kind-linux-amd64 "https://github.com/kubernetes-sigs/kind/releases/download/v0.11.1/kind-linux-amd64" \
-            && chmod +x ./kind-linux-amd64 \
-            && sudo mv ./kind-linux-amd64 /usr/local/bin/kind
       - run:
           name: Install helm
           command: |
             wget https://get.helm.sh/helm-v3.7.1-linux-amd64.tar.gz && \
             tar -zxvf helm-v3.7.1-linux-amd64.tar.gz && \
             chmod +x linux-amd64/helm && \
             sudo mv linux-amd64/helm /usr/local/bin/helm
+  install-kubectl:
+    steps:
       - run:
           name: Install kubectl
           command: |
             curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
             chmod +x kubectl && \
             sudo mv kubectl /usr/local/bin/kubectl
+  install-minikube:
```

Comment:
Add ARCH, and then use this `install-minikube` command whereever `etc/testing/circle/install.sh` and then remove the install minikube step from that file.